eve/tests/single-node/init.sh

```diff
@@ -15,5 +15,5 @@ iptables -F || die
 iptables -X || die
 
 yum install -y epel-release
-yum install -y python2-pip python2-devel python36 python36-devel
-pip2 install tox
+yum install -y python36 python36-devel
```

Comment:
Since `basepython` is removed, `python` will be used which is Python 2. Any reason to then install `python36`, `python36-devel` and `pip3 install tox` instead of keeping the original `python2`-based mechanism?

Reply:
because then it's run on python2, which I don't like...(I make a difference between trying to support python2 and bailing out if we don't support python2 because the CI use it). But anyway python36 doesn't come with pip3.6 neither the pip module (may be a bug on centos package). There is `ensurepip` but I'm not huge fan. Wondering if it's better to keep as now ;)